### ISSUE TITLE
Next round R2

### ISSUE THREAD

**Issue by buck0land** [codetodesign/NYUAMS#692]
Hi guys I have made a new tag R2 for the next round of work.

leo take a look and let me know what you think

-Andrew

**Comment by leobardoramon** [comment]
@barryross , I've assigned 7 new issues to you.

Can you please take a look and provide estimates where they have not yet been provided?

ty.

**Comment by barryross** [comment]
@leobardoramon  yes, will do.  Will provide estimate(s) over this weekend for ones that I haven't yet.

**Comment by leobardoramon** [comment]
@barryross , of the ~actual~ issues marked as "R2" , I have marked the as follows:

5 - fixing
4 - _need_wireframe
--
9 - TOTAL

can you please start putting together (1) vetting fees and (2) quotes for issues marked as "R2"...focus on the issues marked as "fixing" < (*) as I compile wireframes for the 4 remaining issues, I will (a) change labels and (b) update you >

**Comment by leobardoramon** [comment]
@barryross , you can use this link
https://github.com/codetodesign/NYUAMS/issues?q=is%3Aissue+is%3Aopen+label%3A1.5fixing

**Comment by barryross** [comment]
@leobardoramon Yup, will do...thank you keeping it all so organized.  Much appreciated!

**Comment by leobardoramon** [comment]
@buck0land , @gosha1128 

@barryross is expecting a deadline of May 11 , This seems realistic to me.

Thoughts?

**Comment by leobardoramon** [comment]
sorry, guys, 

Barry will need to re-estimate for #661 , due to new parameters

**Comment by leobardoramon** [comment]
... @barryross , could you possibly provide us with a deadline for DEVN-0005 - excluding #661 ?

**Comment by barryross** [comment]
@leobardoramon apologies again for the delay here.  Yes, I can get an updated deadline for you guys by some point tomorrow, excluding #661

**Comment by barryross** [comment]
@leobardoramon Hey Leo, I think by the end of April should be do-able for DEVN-0005 excluding #661.  That being said, I'm refraining from including #693 and #695 in that estimate b/c I am still working on gauging scale/scope.  I do feel a bit blocked moving forward on a number of the current issues...when you can, can you please have a look at the comments/questions in GH?  I'll make sure I've assigned them to you as well...I think I forgot to do that!

thanks!
Barry

**Comment by leobardoramon** [comment]
Yes , @barryross , I understand . I'm going over GH issues now.

**Comment by barryross** [comment]
@leobardoramon great, thanks!

**Comment by leobardoramon** [comment]
@barryross , I know you're deep in code... can you be available this week for a quick chat?

I would like to give @nyuams a status update by Friday of this week
